src/functions/getConfiguration/domain/config.model.ts

```diff
@@ -2,6 +2,7 @@ export interface Config {
   googleAnalyticsId: string;
   approvedDeviceIdentifiers: string[];
   timeTravelDate?: string;
+  role: string;
```

Comment:
Could we have a enum of roles , instead of relying on strings

Reply:
Role enum seems to be a nice improvement here imho as well - more type safety + we don't need to care about case-sensitivity etc